### ISSUE TITLE
HbS hemoglobin

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7201]
**Preferred term label:**
HbS hemoglobin

**Synonyms**
Sickle cell hemoglobin

**Definition (free text, please give PubMed ID)**

Sickle cell hemoglobin (HbS) is caused by a mutation that replaces glutamic acid at residue 6 in beta-globin with valine, which encodes an abnormal form of hemoglobin which is prone to polymerization when exposed to low oxygen tension which results in sickling of red blood cells, hemolytic anemia, vascular occlusion of small vessels, ischemic tissue and organ injury and recurrent painful crises.
PMID:34406727

**Parent term (use hpo.jax.org/app)**
Abnormal hemoglobin HP:0011902

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
SSD and see also https://github.com/TheJacksonLaboratory/loinc2hpoAnnotation/issues/112

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
already existed as HP_0045047
